ui/src/app/applications/components/applications-list/applications-list.tsx

```diff
@@ -318,12 +328,29 @@ export const ApplicationsList = (props: RouteComponentProps<{}>) => {
         );
     }
 
+    function getPageTitle(view: string) {
+        switch (view) {
+            case AppsListViewKey.List:
+                return 'Applications List';
+            case AppsListViewKey.Tiles:
+                return 'Applications Tiles';
+            case AppsListViewKey.Summary:
+                return 'Applications Summary';
+        }
+        return '';
```

Comment:
Same as the other `getPageTitle` function.

Reply:
See response for `getPageTitle`